pages/profile/index.js

```diff
@@ -0,0 +1,55 @@
+import MessageCard from "@/components/MessageCard/MessageCard";
+import ArrowButton from "@/components/ArrowButton/ArrowButton";
+import Profile from "@/components/Profile/Profile";
+import ProfileForm from "@/components/Profile/ProfileForm";
+import { signIn, useSession } from "next-auth/react";
+import Loading from "@/components/Loading/Loading";
+import { useData } from "@/lib/useData";
+import { useState } from "react";
+import { formatedUserInfo } from "@/lib/profile/profileHelper";
+
+export default function profilePage() {
```

Comment:
Please use PascalCase for component names.
```suggestion
export default function ProfilePage() {
```